### ISSUE TITLE
Cueing arrow can jump around 

### ISSUE THREAD

**Issue by KatieWoe** [phetsims/mean-share-and-balance#350]
**Test device**
Samsung
**Operating System**
Win 11
**Browser**
Chrome
**Problem description**
For https://github.com/phetsims/qa/issues/1135
The cueing arrow, if moved by keyboard nav, will jump back to it's original position if you click on a soccer ball. This was not in rc.1
**Steps to reproduce**
1. Kick all 7 soccer balls
2. Use keyboard nav shift focus to a soccer ball that doesn't have the cueing arrow. Do not pick it up
3. Tab to take focus off the soccer ball area
4. Click a soccer ball, without dragging

**Visuals**
![oddcuejump](https://github.com/user-attachments/assets/bdc0fb6b-bd32-4858-9ae6-f2fafd185e86)

Troubleshooting information:

<details>

!!!!! DO NOT EDIT !!!!!
Name: ‪Mean: Share and Balance‬
URL: https://phet-dev.colorado.edu/html/mean-share-and-balance/1.1.0-rc.3/phet/mean-share-and-balance_all_phet.html
Version: 1.1.0-rc.3 2024-08-20 21:01:51 UTC
Features missing: applicationcache, applicationcache, touch
Flags: pixelRatioScaling
User Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/127.0.0.0 Safari/537.36
Language: en-US
Window: 1536x695
Pixel Ratio: 1.25/1
WebGL: WebGL 1.0 (OpenGL ES 2.0 Chromium)
GLSL: WebGL GLSL ES 1.0 (OpenGL ES GLSL ES 1.0 Chromium)
Vendor: WebKit (WebKit WebGL)
Vertex: attribs: 16 varying: 30 uniform: 4096
Texture: size: 16384 imageUnits: 16 (vertex: 16, combined: 32)
Max viewport: 32767x32767
OES_texture_float: true
Dependencies JSON: {}

</details>

**Comment by Nancy-Salpepi** [comment]
this looks fixed in rc.4. 
Closing.
